src/nidigital/metadata/functions.py

```diff
@@ -2048,11 +2769,14 @@
     'Initiate': {
         'codegen_method': 'private',
         'documentation': {
-            'description': 'Starts bursting the pattern configured by NIDIGITAL_ATTR_START_LABEL,\ncausing the NI-Digital sessionto be committed. To stop the\npattern burst, call niDigital_Abort. If keep alive pattern is\nbursting when niDigital_Abort is called or upon exiting the\ncontext manager, keep alive pattern will not be stopped. To\nstop the keep alive pattern, call niDigital_AbortKeepAlive.'
+            'description': 'Starts bursting the pattern. If you do not call the niDigital_Commit function, this function implicitly calls the niDigital_Commit function for you.\n'
```

Comment:
This overwrites my change in #1406 as we thought it would. Since the process is manual anyway, do we want to manually not overwrite it here. I am going to change the internal metadata to keep them in sync.

Reply:
I think I restored the previous text in the latest commit.